TokenAdministrationApi.Tests/V1/Gateways/TokensGatewayTests.cs

```diff
@@ -5,27 +5,95 @@
 using FluentAssertions;
 using TokenAdministrationApi.V1.Gateways;
 using NUnit.Framework;
+using TokenAdministrationApi.V1.Infrastructure;
 using TokenAdministrationApi.V1.Boundary.Requests;
+using TokenAdministrationApi.Tests.V1.Helper;
+using TokenAdministrationApi.V1.Domain;
+using System.Collections.Generic;
 
 namespace TokenAdministrationApi.Tests.V1.Gateways
 {
-    //For instruction on how to run tests please see the wiki: https://github.com/LBHackney-IT/lbh-TokenAdministrationApi/wiki/Running-the-test-suite.
     [TestFixture]
     public class TokensGatewayTests : DatabaseTests
     {
         private readonly IFixture _fixture = new Fixture();
         private TokensGateway _classUnderTest;
-
         [SetUp]
         public void Setup()
         {
             _classUnderTest = new TokensGateway(DatabaseContext);
         }
 
+        [Test]
+        public void ShouldGetAllTokensFromDatabase()
+        {
+            var token = AuthTokenDatabaseEntityHelper.AddTokenRecordToTheDatabase(null, DatabaseContext);
+            var disabledToken = AuthTokenDatabaseEntityHelper.AddTokenRecordToTheDatabase(false, DatabaseContext);
+            var result = _classUnderTest.GetAllTokens(null);
+
+            result.Should().NotBeNull();
+            result.Count.Should().Be(2);
+            result.Find(x => x.Id == token.Id).ApiEndpointName.Should().Be(token.ApiEndpointName);
```

Comment:
Maybe extract `result.Find(x => x.Id == token.Id)` into it's a variable to tidy this up a bit.